e2e/test_reconnection.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+
+case "$1" in
+  rest)
+    rest_arg="--rest"
+    server_app_address="http://localhost:9091"
+    server_address="http://localhost:9093"
+    db_arg="--database :flwr-in-memory-state:"
+    ;;
+  sqlite)
+    rest_arg=""
+    server_address="127.0.0.1:9092"
+    server_app_address="127.0.0.1:9091"
+    db_arg="--database $(date +%s).db"
+    ;;
+  *)
+    rest_arg=""
+    server_address="127.0.0.1:9092"
+    server_app_address="127.0.0.1:9091"
+    db_arg="--database :flwr-in-memory-state:"
+    ;;
```

Comment:
Do we need this part if in `e2e.yml`, we only have `./../test_reconnection.sh sqlite`?

Reply:
It was in case we want to extend the test in the future (currently `in_memory` state won't work)